api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirConceptDaoImpl.java

```diff
@@ -70,4 +72,30 @@ public Optional<Concept> getConceptWithSameAsMappingInSource(@Nonnull ConceptSou
 		
 		return Optional.ofNullable((Concept) criteria.uniqueResult());
 	}
+	
+	@Override
+	public List<Concept> getConceptsWithAnyMappingInSource(@Nonnull ConceptSource conceptSource,
```

Comment:
Can you add a test or two for this DAO method? Then I think this is good to merge.

Reply:
Thanks @ibacher , had forgoten to add tests for that one .
added